### ISSUE TITLE
RSS Dashboard menu item no longer in the Help menu

### ISSUE THREAD

**Issue by twasink** [spring-projects/spring-ide#208]
The menu item to open the RSS Dashboard (with the buttons to do things such as manage the IDE extensions, etc) is no longer in the Help menu, as of 3.9.0.

There is a button in the toolbar to open it, but that's nowhere near as discoverable as the menu item.

**Comment by spring-issuemaster** [comment]
(comment in Pivotal Tracker added by Kris De Volder:) 

I found where this was removed and put it back in. 

However, it seems like it was removed during a drive to supply high-resolution icons for everything. Presumably because the high-res icon support didn't work here or something

I don't have a display that benefits from retina-style icons so the low-res icon looks fine to me, but it may look 'fuzzy' for the retina folks.

Someone with a Mac should probably give it a try to see how bad it looks.
